addons/mail/static/src/discuss/web/chat_window_patch.xml

```diff
@@ -3,7 +3,7 @@
     <t t-inherit="mail.ChatWindow" t-inherit-mode="extension">
         <xpath expr="//PinnedMessagesPanel" position="after">
             <CallSettings t-elif="state.activeMode === 'in-settings'" thread="thread" className="'flex-grow-1'"/>
-            <ChannelMemberList t-elif="state.activeMode === 'member-list'" thread="thread" className="'flex-shrink-0 border-start w-100 h-100'"/>
+            <ChannelMemberList t-elif="state.activeMode === 'member-list'" thread="thread" className="'flex-shrink-0 w-100 h-100'"/>
             <ChannelInvitation t-elif="state.activeMode === 'add-users'" thread="thread" chatState="state"/>
```

Comment:
`w-100 h-100` for `ChannelInvitation` as well? I feel like maybe in the chat window we need to reposition and reduce some unimportant info since the space is pretty narrow.

Reply:
The visual of `ChannelInvitation` is good at the moment. It's not if we add `w-100 h-100`, because the content is less than the chat window size:

Before / After
<img width="347" alt="Screenshot 2023-05-26 at 11 46 18" src="https://github.com/odoo/odoo/assets/6569390/b8f51679-70a3-4d9c-b307-cc7613288ab1"> <img width="339" alt="Screenshot 2023-05-26 at 11 46 34" src="https://github.com/odoo/odoo/assets/6569390/045817c0-2f05-4a38-9d78-5186b9a41b55">

`ChannelMemberList` has `w-100 h-100` because this component handles sizing of content properly, with scrollable too.

So it's best to not make this change.